archive_verify/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "1.0.0"
+__version__ = "1.0.1"
```

Comment:
It looks like this version tag is a bit outdated: https://github.com/Molmed/snpseq-archive-verify/releases

Reply:
Good catch! Fixed now.